pkg/components/util/helm.go

```diff
@@ -201,12 +201,6 @@ func chartFromManifests(name string, manifests map[string]string) (*chart.Chart,
 				continue
 			}
 
-			// Drop Namespace resource as we take care of its creation at another level and we don't want resources to collide.
```

Comment:
This commit has 2 different changes. One is to disable filtering of Namespace resource in the charts and if we do that, we need to make sure that all components have no Namespace object defined.

Other change is to use Helm built-in functionality to create release namespace, which I agree we should use and I already do that in another PR. Also with that, there is more code which needs to be touched. See https://github.com/kinvolk/lokomotive/pull/628/commits/bdfac344d64eb27852e1d3d2ea95c484a7e3c18a.

Reply:
Done.